### PR TITLE
Week-4 added the Wechaty environment variables

### PR DESCRIPTION
**Pull request by shraddhavp**
This PR contains the details about the environment variable in Wechaty
Will be adding more environment variables as soon as I get more information about it 
Link #890

### CODE REVIEW

**Review by Rohitesh-Kumar-Jain — COMMENTED** [review]
I am not the best person to comment on the environment variables and their description, document structure seems good to me.

**Comment by Rohitesh-Kumar-Jain** [comment]
I am approving this PR for style and I have proofread this, someone from the org should verify that the content of the doc is correct or not.

**Comment by shraddhavp** [comment]
> I am approving this PR for style and I have proofread this, someone from the org should verify that the content of the doc is correct or not.

yes sure @Rohitesh-Kumar-Jain  thank you

File: docusaurus/docs/specs/wechaty.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
 ---
-title: Polyglot Wechaty
+title: Wechaty Environment Variables
 sidebar_label: Wechaty
 ---
 
-Polyglot Wechaty is ... (tbw)
+Wechaty has different environment variables that need to be set for successful implementation of the bot.This section is dedicated to describe  the various envirnoment variables provided by Wechaty.
```
Comment (proudofsimin) [inline docusaurus/docs/specs/wechaty.md]: Wechaty requires different environment variables to implement the bot. In this section, you will learn how to use the various environment variables.

How do you think if we change the sentence to be more direct?

**Review by proudofsimin — APPROVED** [review]
I like the table. Thank you

File: docusaurus/docs/specs/wechaty.md (hunk 1 above)
Comment (shraddhavp) [reply docusaurus/docs/specs/wechaty.md]: yes will do so @proudofsimin

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Perfect PR : clean commit history, properly based, PR good in scope.

**Review by huan — APPROVED** [review]
LGTM.

Thanks for improving the environment variables docs and adding this beautiful table for easy reading!

**Comment by shraddhavp** [comment]
@lijiarui mam can you please check this and merge it too?Huan sir has also approved it

**Comment by lijiarui** [comment]
hi @shraddhavp As GSoD Project PR Review Rules in https://github.com/wechaty/wechaty.js.org/issues/1045

This PR should follow our rules and get enough approval, then I will review this PR. So I remove the `ready to merge` tag and feel free to add this tag when it gets enough approval.

**Comment by shraddhavp** [comment]
> hi @shraddhavp As GSoD Project PR Review Rules in #1045
> 
> This PR should follow our rules and get enough approval, then I will review this PR. So I remove the `ready to merge` tag and feel free to add this tag when it gets enough approval.

mam @lijiarui  but this PR has got 3 approvals right? any other changes I should make ?

**Comment by lijiarui** [comment]
> > hi @shraddhavp As GSoD Project PR Review Rules in #1045
> > This PR should follow our rules and get enough approval, then I will review this PR. So I remove the `ready to merge` tag and feel free to add this tag when it gets enough approval.
> 
> mam @lijiarui but this PR has got 3 approvals right? any other changes I should make ?

You should got the following approval as #1045 showed

## For writers:
Writers who are in the same team are **REQUIRED** to review each other’s PR, and give “Request changes” or “Approve”
Writers are encouraged to give “Request changes”, “Approve”, or “Comment” to any PRs. (please contribute to the whole team by doing it!)

Your team member is @Soumi7, so you should git @Soumi7 approval first.

## For volunteers:
Volunteers are **REQUIRED** to review all PRs created by writers and give “Request changes” or “Approve”

We have two volunteers, so you should both got @Rohitesh-Kumar-Jain and @proudofsimin approval, there is no problem for this part.

**Comment by shraddhavp** [comment]
> > > hi @shraddhavp As GSoD Project PR Review Rules in #1045
> > > This PR should follow our rules and get enough approval, then I will review this PR. So I remove the `ready to merge` tag and feel free to add this tag when it gets enough approval.
> > 
> > 
> > mam @lijiarui but this PR has got 3 approvals right? any other changes I should make ?
> 
> You should got the following approval as #1045 showed
> 
> ## For writers:
> Writers who are in the same team are **REQUIRED** to review each other’s PR, and give “Request changes” or “Approve”
> Writers are encouraged to give “Request changes”, “Approve”, or “Comment” to any PRs. (please contribute to the whole team by doing it!)
> 
> Your team member is @Soumi7, so you should git @Soumi7 approval first.
> 
> ## For volunteers:
> Volunteers are **REQUIRED** to review all PRs created by writers and give “Request changes” or “Approve”
> 
> We have two volunteers, so you should both got @Rohitesh-Kumar-Jain and @proudofsimin approval, there is no problem for this part.

ok mam

**Comment by Rohitesh-Kumar-Jain** [comment]
@Soumi7 We would be glad if you may once go through the PR, and either request for changes or approve the PR.

**Review by Soumi7 — APPROVED** [review]
LGTM

**Comment by shraddhavp** [comment]
@lijiarui  mam can you please review the PR and give us the feedback?

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by lijiarui — APPROVED** [review]

**Pull request merged**